InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -53,7 +52,7 @@
     @Override
     public void robotInit() {
         config = ConfigChooser.getConfig();
-        DriveConstants.setConfig(config);
+        DriveUtils.setConfig(config);

```

Comment:
This is better, but does the DriveUtils class have to be static? If it requires an instance variable (config), why not just make it a member variable of the DriveSubsystem?

Reply:
good point, I used static because DriveConstants was static but I guess it's not needed